widgetastic_manageiq/__init__.py

```diff
@@ -5946,3 +5946,17 @@ def messages(self):
         except NoSuchElementException:
             pass
         return result
+
+
+class SSUICardPFInfoStatus(Widget):
```

Comment:
Consider subclassing `widgetastic.widget.GenericLocatorWidget` when setting up a widget like this, as it takes care of mapping ROOT to the passed locator.